Ginger/GingerCoreNET/Reports/GingerReport.cs

```diff
@@ -211,7 +222,18 @@ public int TotalBusinessFlows
         {
             get
             {
-                return BusinessFlowReports.Count();
+                if (BusinessFlowReports != null && BusinessFlowReports.Count > 0)
+                {
+                    return BusinessFlowReports.Count();
+                }
+                else if (GingerRunner != null)
+                {
+                    return GingerRunner.BusinessFlows.Where(x => x.Active == true).ToList().Count();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5700269)